packages/fxa-settings/src/lib/oauth/hooks.tsx

```diff
@@ -283,3 +285,37 @@ export function useFinishOAuthFlowHandler(
   }
   return { oAuthDataError: null, finishOAuthFlowHandler };
 }
+
+/**
+ * If we don't have `keyFetchToken` or `unwrapBKey` and it's required, the user needs to
+ * restart the flow. We only store these values in memory, so they don't persist across page
+ * refreshes or browser session restarts. We can't redirect to /signin and reprompt for
+ * password in a browser session restart/crash because the browser stores the flow state in
+ * memory and the query params won't match after a browser session is restored.
+ *
+ * TODO: Can we check session storage for if the user refreshed so we can redirect them
```

Comment:
Have we had input from UX on how to display this error? Or have a ticket filed to review UX? I know it's a edge case, but I don't love that we display a screen with nothing but an error message and no path forward.

Reply:
Mark and I agreed on this for now, and also agreed it wasn't the best. I did note that reset PW refresh in a comment here because I'm thinking we can bump up the points to a 3 and use that ticket to handle refreshes better here as well, but if a browser crash happens, the user really needs to just restart the flow from the profile menu.